index.html

```diff
@@ -62,6 +62,157 @@ <h1>
 
         </div>
     </section>
+    <!-- Works -->
+    <section class="works">
+        <div class="works_content">
+            <h1>My Recent Works</h1>
+            <img src="Assets/Indicator.png" alt="">
+        </div>
+        <div class="works_cards">
+            <div class="card">
+                <div class="card_image">
+                    <img src="" alt="">
+                </div>
+                <div class="card_content">
+                    <h1>Multi-Post Stories <br> Gain+Glory</h1>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="" alt="">
```

Comment:
As I previously mentioned, please follow the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) and [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=xw0Xo7v32ZLKEBS9-0) design